src/v2/core.ts

```diff
@@ -78,32 +78,14 @@ interface CloudEventBase<T> {
 
   /** Information about this specific event. */
   data: T;
-
-  /**
-   * A map of template parameter name to value for subject strings.
-   *
-   * This map is only available on some event types that allow templates
-   * in the subject string, such as Firestore. When listening to a document
-   * template "/users/{uid}", an event with subject "/documents/users/1234"
-   * would have a params of {"uid": "1234"}.
-   *
-   * Params are generated inside the firebase-functions SDK and are not
-   * part of the CloudEvents spec nor the payload that a Cloud Function
-   * actually receives.
-   */
-  params?: Record<string, string>;
```

Comment:
Note: params should probably only be present in Cloud Function types that actually support params.